### ISSUE TITLE
Question: batch_size=1 for Transporter models?

### ISSUE THREAD

**Issue by MohitShridhar** [google-research/ravens#7]
Thanks for open-sourcing this great work!

I noticed that all the Transporter models are trained with a [batch size of 1](https://github.com/google-research/ravens/blob/master/ravens/models/transport.py#L99). Is there anything preventing the use of larger batches other than memory?

**Comment by andyzeng** [comment]
Thanks Mohit! Other than memory, there shouldn't be anything preventing the use of larger batches. Note that the [input rotations](https://github.com/google-research/ravens/blob/master/ravens/models/transport.py#L109) expand the batch size to the number of rotations (default 36). So a batch size of 2 should be expanded to 2 * n_rotations. To support this, you may need to make a few changes to the data preprocessing and loss function.

**Comment by MohitShridhar** [comment]
Awesome, thanks!

**Comment by MohitShridhar** [comment]
Hi Andy, sorry, just a quick follow-up: for the evaluations in Table 2 of the paper, were the agents trained for a fixed number of iterations (1k? 10k?). Or did you pick the agents with the best validation loss for each run?

**Comment by andyzeng** [comment]
Hi Mohit, I believe we trained the agents for multiple fixed iterations (1K, 2K, 5K, 10K, 20K, 40K), then picked the iteration with the best average validation loss between seeds.
